queries/dune_v2/period_slippage.sql

```diff
@@ -1,7 +1,30 @@
 -- https://github.com/cowprotocol/solver-rewards/pull/327
--- Query Here: https://dune.com/queries/3093726
+-- Query Here: https://dune.com/queries/3333368
 with
-batch_meta as (
+block_range_start as (
+  select
+    max("number") + 1 as start_block
+  from
+    ethereum.blocks
+where
+      time <= cast('{{StartTime}}' as timestamp)
+)
+,block_range_stop as (
+    select
+      max("number") as end_block
+    from
+      ethereum.blocks
+    where
+      time <= cast('{{EndTime}}' as timestamp)
+  )
+,block_range as (
+    select
+      *
+    from
+      block_range_start
+      cross join block_range_stop
+  )
```

Comment:
Similar to the comment above, one could do
```suggestion
,block_range as (
    select
        min("number") as start_block,
        max("number") as end_block
    from ethereum.blocks
    where time >= cast('{{StartTime}}' as timestamp) and time < cast('{{EndTime}}' as timestamp)
  )
```
or with other inequalities for including different bounds.